setup.cfg

```diff
@@ -55,6 +55,3 @@ test =
     nbval>=0.10.0
 simstore =
     # simstore requirements have been moved to main
-
-[bdist_wheel]
-universal = 1
```

Comment:
Why was this removed?

Reply:
Turns out I'd misremembered what `bdist_wheel.universal` does.  It isn't the equivalent of noarch; it is noarch + compatible with Python 2 and 3. We're no longer compatible with Python 2. With `universal = 1`, wheels claim to be Python 2 compatible, even if `python_requires` is >3 (e.g., https://pypi.org/project/openpathsampling/1.6.0/#files)

I probably knew that when I first added that to a project, but kinda forgot.